### ISSUE TITLE
加载model作为预训练模型时出现与训练网络不匹配的问题。

### ISSUE THREAD

**Issue by liufangcen** [erfect2020/DecompositionForFusion#2]
您好！您的图像融合工作非常吸引人，因此我对代码进行了详细的学习，但是目前存在两个问题困扰了我很久，希望得到作者的帮助！

1. 我想用您提供的model进一步训练网络。我首先修改了训练的yaml文件，然后对当前的模型参数与预训练的模型参数进行了对比筛选。当我运行上代码的时候发现您提供的model中的参数与训练网络中的参数不匹配。
我尝试过自己去解决这个问题，比如说加入了commom_part= torch.cat((feature_commom,features_upper3,feature_lower3),dim=1，然而运行仍然有问题。
![image](https://user-images.githubusercontent.com/43163498/220133741-1bbc41a7-6513-4ecd-b47e-d69c5c69ee0e.png)

2. 我在用您提供的model测试RoadScene数据集时，所测出的ssim指标为：1.4366，无法得到与论文中所提及的指标0.727。
我直接下载了数据集，并且网络结构没有做任何改动。不知道是否是有什么需要注意的地方我忽略了？
![image](https://user-images.githubusercontent.com/43163498/220132806-1cd45034-dc2a-40d6-87f6-f6d265b2afbd.png)

希望作者能够百忙之中抽出时间回复一下！

非常感谢。

**Comment by erfect2020** [comment]
谢谢你的关注和支持。
问题1:训练代码model超参写错了。
问题2:1.4366需要除以2. 单图SSIM~(0,1).

**Comment by liufangcen** [comment]
非常感谢您及时的回复！

请问您是否能够提供一下正确的model参数呀？

**Comment by erfect2020** [comment]
https://github.com/erfect2020/DecompositionForFusion/commit/d957f64889dd455832d8b4010dd8363b99daac01
